l10n_br_fiscal/models/document_fiscal_mixin_methods.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (C) 2019  Renato Lima - Akretion <renato.lima@akretion.com.br>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, models
+
+
+class FiscalDocumentMixin(models.AbstractModel):
```

Comment:
Tem que mudar o nome dessa class:
```suggestion
class FiscalDocumentMixinMethods(models.AbstractModel):
```